### PR TITLE
Wall frames color fix

### PR DESCRIPTION
**Pull request by Terror4000rus**
Fixed color of wall frames. Not hull one (because a mapper have to do so).

### CODE REVIEW

File: code/game/objects/structures/wall_frame.dm [hunk 1]
```diff
@@ -137,6 +136,9 @@
 /obj/structure/wall_frame/hull
 	color = COLOR_HULL
 
+/obj/structure/wall_frame/prepainted
+	color = COLOR_GUNMETAL
```
Comment (JakeDaBoss) [inline code/game/objects/structures/wall_frame.dm]: Why is this necessary? Wallframes aren't meant to be colored. They do not act like walls in the sense that they should be painted.

Also, we don't use painted versions of the wall currently.

**Review by ghost — COMMENTED** [review]
If you can make this aspect of your fix a separate PR, we can get it merged, the other one may need a further look at from the team.

File: code/game/objects/structures/wall_frame.dm (hunk 1 above)
Comment (Terror4000rus) [reply code/game/objects/structures/wall_frame.dm]: And that's why I made this subtype. If someone want to use prepainted walls, they will use this. If not - previous type (which is already used at the ship and colony, but has own color while walls don't).

Comment (JakeDaBoss) [reply code/game/objects/structures/wall_frame.dm]: We won't ever use it. Ishimura won't use it, neither will Tau. Sprawl is a long shot for similar reasons. It seems you added it because you think/hope they'll be used, but I can tell you now, they won't. I've spoken to Snype about this multiple times.

Comment (Terror4000rus) [reply code/game/objects/structures/wall_frame.dm]: It looks a little dumb. You get mad about one new suitable type (because there already other type with same color in code) and hold the whole PR because of it. Good mappers have their own vision on their job and may see situations where current color will be better than regular blank walls.

**Comment by Terror4000rus** [comment]
> If you can make this aspect of your fix a separate PR, we can get it merged, the other one may need a further look at from the team.

...Alright. I'll make other branch for magazines tweak. This is so ... , really.

File: code/game/objects/structures/wall_frame.dm (hunk 1 above)
Comment (JakeDaBoss) [reply code/game/objects/structures/wall_frame.dm]: I'm not mad, I'm saying it's wasted effort and time.

**Comment by Terror4000rus** [comment]
I will add examples of poor mapping and poor mapping ahead, just to end this pointless conversation conversation.
Upd: wrong, I don't have a reason to up the server than for it.

File: code/game/objects/structures/wall_frame.dm [hunk 2]
```diff
@@ -137,6 +136,9 @@
 /obj/structure/wall_frame/hull
 	color = COLOR_HULL
 
+/obj/structure/wall_frame/prepainted
```
Comment (JakeDaBoss) [inline code/game/objects/structures/wall_frame.dm]: Not needed. The rest is a decent addition.

**Review by JakeDaBoss — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/wall_frame.dm (hunk 2 above)
Comment (Terror4000rus) [reply code/game/objects/structures/wall_frame.dm]: ...You already said this opinion before. People aren't so stupid to don't understand what you wanted to say about it.

Comment (JakeDaBoss) [reply code/game/objects/structures/wall_frame.dm]: The previous was a comment, this was me actually opposing it. There's a way GitHub works, you know.

**Comment by Terror4000rus** [comment]
I appreciate my time to don't hold horns, @WestHybrid 
Just hope mappers will have more skill than one who made the ship with such wall frames. The bridge windows still make me sick.

**Comment by Snypehunter007** [comment]
@Terror4000rus if you have mapping suggestions, other than the two you've already made, feel free to post them here on the Github.

Also, just to make sure I'm on the same page, what exactly does this change do?

**Comment by JakeDaBoss** [comment]
> @Terror4000rus if you have mapping suggestions, other than the two you've already made, feel free to post them here on the Github.
> 
> Also, just to make sure I'm on the same page, what exactly does this change do?

Removes color from the current wallframes. Which should be promptly denied, unless we want discoloration with respect to walls.

**Comment by Terror4000rus** [comment]
Sorry, but are you blind?

**Comment by Terror4000rus** [comment]
I rechecked and something strange appeared, but now, it has to be better.
Before:
<details>

image


</details>
After:
<details>

image


<details>

**Comment by NanakoAC** [comment]
I'm not quite seeing the difference in the above pictures, can you explain a bit more whats going on here?

**Comment by Terror4000rus** [comment]
> I'm not quite seeing the difference in the above pictures, can you explain a bit more whats going on here?

Wall frames in lower picture have almost the same color as walls. One above - don't.

**Review by NanakoAC — APPROVED** [review]
i see now, its subtle, but good change
